src/include/rbd/librbd.h

```diff
@@ -144,6 +144,8 @@ enum {
   RBD_IMAGE_OPTION_JOURNAL_ORDER = 5,
   RBD_IMAGE_OPTION_JOURNAL_SPLAY_WIDTH = 6,
   RBD_IMAGE_OPTION_JOURNAL_POOL = 7,
+  RBD_IMAGE_OPTION_FEATURES_SET = 8,
```

Comment:
yes MASK can work in my example. but that's not a general solution I think. It can only clear bits in default features but can not set bits in features which is the reason I introduce FEATURE SET